docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -695,7 +695,7 @@ Sample multistage Dockerfile for building an image from `scratch`:
 [source,dockerfile,subs=attributes+]
 ----
 ## Stage 1 : build with maven builder image with native capabilities
-FROM quay.io/quarkus/ubi-quarkus-mandrel-builder-image:{mandrel-flavor} AS build
+FROM quay.io/quarkus/ubi-quarkus-graalvmce-builder-image:{graalvm-flavor} AS build
```

Comment:
@cescoffier I don't see a https://quay.io/repository/quarkus/ubi-quarkus-graalvmce-builder-image?tab=tags for 23.0 so I don't think this will fly?

Reply:
Yes, that's normal, it does not exist - new scheme, it's named `jdk-17.0.7` - that's typically why we need to read the full documentation and make sure the right variable/version is used everywhere.

Reply:
@cescoffier hmmm maybe I'm blind but that I checked. I don't see a `jdk-17` one either for `graalvmce`.

Reply:
For testing I used `22.3`, I can notice that they apparently changed the version scheme, and we are relying on a ["legacy" version scheme used by mandrel](https://github.com/graalvm/setup-graalvm/pull/46/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R30).

But the `quarkus-images` are already built [using the new scheme](https://github.com/quarkusio/quarkus-images/commit/2523f8aa136040c58f1addd75fe828e1c98adebf#diff-f4027079cb28e2c5c0812436633b5db6bb463cde364a515ab2612889725a7331).

Let me know if I should do anything to move this forward 🙂

Reply:
The new version scheme images have been released correctly: `jdk-17.0.7-amd64`, `jdk-20.0.1-amd64`

Reply:
There are not on the first page...

Reply:
@gsmet you can find them using the "Filter Tags..." input box on the right. (GH doesn't allow to upload an image at the moment 😞 )